frontend/src/components/projects/ProjectsAssociatedSamples.js

```diff
@@ -0,0 +1,168 @@
+    {
+      title: "Sample Kind",
+      dataIndex: "sample_kind__name",
+      sorter: true,
+      width: 70,
+      options: sampleKinds.items.map(x => ({ label: x.name, value: x.name })), // for getFilterProps
+      render: (_, sample) =>
+        <Tag>{sampleKinds.itemsByID[sample.sample_kind].name}</Tag>,
+    },
+    {
+      title: "Name",
+      dataIndex: "name",
+      sorter: true,
+      width: 170,
+      render: (name, sample) =>
+        <Link to={`/samples/${sample.id}`}>
+          <div>{name}</div>
+          {sample.alias &&
+            <div><small>alias: {sample.alias}</small></div>
+          }
+        </Link>,
+    },
+    {
+      title: "Cohort",
+      dataIndex: "individual__cohort",
+      sorter: true,
+      width: 130,
+      render: (_, sample) => {
+        const individual = sample.individual
+        return (individual &&
+          <Link to={`/individuals/${individual}`}>
+            {withIndividual(individualsByID, individual, individual => individual.cohort, "loading...")}
+          </Link>)
+      }
+    },
+    {
+      title: "Vol. (µL)",
+      dataIndex: "volume",
+      sorter: true,
+      align: "right",
+      className: "table-column-numbers",
+      width: 70,
+    },
+    {
+      title: "Depleted",
+      dataIndex: "depleted",
+      sorter: true,
+      render: depleted => <SampleDepletion depleted={depleted} />,
+      width: 50,
+    }
+  ];
+
+const mapStateToProps = state => ({
+  token: state.auth.tokens.access,
+  sampleKinds: state.sampleKinds,
+  page: state.samples.page,
+  samples: state.samples.items,
+  samplesByID: state.samples.itemsByID,
+  individualsByID: state.individuals.itemsByID,
+  totalCount: state.samples.totalCount,
+  isFetching: state.samples.isFetching,
+  filters: state.samples.filters,
+  sortBy: state.samples.sortBy,
+});
+
+const actionCreators = {listTable, setFilter, clearFilters, setSortBy};
+
+const ProjectsAssociatedSamples = ({
+  token,
+  projectID,
+  samples,
+  samplesByID,
+  individualsByID,
+  sampleKinds,
+  isFetching,
+  page,
+  filters,
+  totalCount,
+  sortBy,
+  listTable,
+  setFilter,
+  clearFilters,
+  setSortBy,
+}) => {
+
+  useEffect(() => {
+    //will be called on component will mount
+    clearFilters()
+    // returned function will be called on component unmount
+    return () => {
+      clearFilters()
+    }
+  }, [])
+
+  setDefaultFilter(SAMPLE_FILTERS.projects__id.key, projectID, setFilter, filters, clearFilters)
+  let {projects, ...filtersForWarning} = filters
+
+
+  const columns = getTableColumns(sampleKinds, individualsByID)
+  .map(c => Object.assign(c, getFilterProps(
+    c,
+    SAMPLE_FILTERS,
+    filters,
+    setFilter,
+    setFilterOption,
+  )))
+
+  const nFilters = getNFilters(filters)
+  const nFiltersForWarning = nFilters - 1
+
+  //To ensure user doesn't see the full list of samples
+  samplesByID = samples.length === totalCount ? samplesByID : {}
```

Comment:
As far as i can see this flushes the whole local cache of samples ... this may be overkill ... When i browse the site if i jump from samples to container and other tabs the sample show up instantly when i come back ... when i load a project list of samples, if i go back to the samples list i get a spinner and wait for the samples to load. I think clearing the samples (items) would be less damaging to the navigation if it makes any sense.